### PR TITLE
add vmware-iso builder

### PR DESCRIPTION
**Pull request by tduehr**

### CODE REVIEW

**Comment by ianchesal** [comment]
Needs a spec test. At a minimum test that it works, doesn't work when required options are present and missing. See https://github.com/ianchesal/packer-config/blob/master/spec/packer/builders/vmware_vmx_spec.rb for an example. Thanks!

**Comment by tduehr** [comment]
Errors corrected and spec added.

**Comment by tduehr** [comment]
and fixed again...

File: spec/packer/builders/vmware_iso_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Encoding: utf-8
+require 'spec_helper'
+
+RSpec.describe Packer::Builder::VMWareISO do
```
Comment (ianchesal) [inline spec/packer/builders/vmware_iso_spec.rb]: You called the class `VmwareISO` in the module.

File: lib/packer/builders/vmware_iso.rb [hunk 2]
```diff
@@ -0,0 +1,206 @@
+# Encoding: utf-8
+require 'packer/builder'
+require 'packer/dataobject'
+
+module Packer
+  class Builder < Packer::DataObject
+    class VmwareISO < Builder
```
Comment (ianchesal) [inline lib/packer/builders/vmware_iso.rb]: Should be 'VMWareISO' instead, maybe?

**Comment by tduehr** [comment]
This one will work... I know it... maybe next time I'll remember to run the specs locally before pushing...

**Comment by ianchesal** [comment]
Can you open this PR against `develop` instead of `master`? If you do that, I'll merge as-is and then fix it and do a new release.

**Pull request closed without merging**